docs/guides/references/advanced-installation.mdx

```diff
@@ -7,6 +7,7 @@ title: Advanced Installation
 | Name                             | Description                                                                                   |
 | -------------------------------- | --------------------------------------------------------------------------------------------- |
 | `CYPRESS_INSTALL_BINARY`         | [Destination of Cypress binary that's downloaded and installed](#Install-binary)              |
+| `CYPRESS_CONNECT_RETRY_THRESHOLD`| Overrides the maximum number of retries that are done when trying to connect to a browser     |
```

Comment:
Can we add the default timeout?

Reply:
Yes, good Idea. I added  the default timeout